tests/python/PyImportSortBearTest.py

```diff
@@ -1,29 +1,93 @@
 from bears.python.PyImportSortBear import PyImportSortBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
-PyImportSortBearTest = verify_local_bear(PyImportSortBear,
-                                         ('import os\nimport sys\n',
-                                          'import os\nimport sys\n'),
-                                         ('import sys\nimport os\n',
-                                          'import sys\nimport os\n'))
-
-PyImportSortBearConfigsTest = verify_local_bear(
-    PyImportSortBear,
-    ('from os import read\nfrom sys import *\n',),
-    ('from os import read\nfrom os import *\n',),
-    settings={'combine_star_imports': True})
```

Comment:
Why are a lot of test classes like `PyImportSortBearConfigsTest` being removed?
It looks like some useful test scenarios are being completely removed..?